### PR TITLE
Add Rander and LogProber to distmv

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

File: distmv/interface.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package distmv
```
Comment (btracey) [inline distmv/interface.go]: Copyright

**Comment by kortschak** [comment]
Some background?

**Comment by btracey** [comment]
I was/am playing around with a possible API for optimize.Global (and the test method needs this). It's across repositories, so it can't go in as one change anyway. If you'd like me to hold off until I have working code, I can do that too.

**Comment by kortschak** [comment]
Having a reason in the commit message or a link to an issue would be good. I can't really review it unless I know what it needs to do.

**Comment by btracey** [comment]
Okay. I'll keep it local for now then.

**Pull request closed without merging**